### ISSUE TITLE
Rule request: Unnecessary parentheses

### ISSUE THREAD

**Issue by dbrgn** [sasstools/sass-lint#1213]
With this file:

```scss
$a: 10px;
$b: 20px;

a {
  width: ($a + $b);
}
```

...sass-lint does not complain about the unnecessary parentheses.

Would this be easy to implement?
